torch/csrc/api/include/torch/nn/functional/loss.h

```diff
@@ -352,6 +352,45 @@ inline Tensor smooth_l1_loss(
 
 // ============================================================================
 
+#ifndef DOXYGEN_SHOULD_SKIP_THIS
+namespace detail {
+inline Tensor huber_loss(
+    const Tensor& input,
+    const Tensor& target,
+    HuberLossFuncOptions::reduction_t reduction,
+    double beta = 1.) {
+  if (target.sizes() != input.sizes()) {
+    TORCH_WARN("Using a target size (", target.sizes(), ") that is different to the input size (", input.sizes(), "). ",
```

Comment:
TORCH_WARN_ONCE (see https://github.com/pytorch/pytorch/wiki/Developer-FAQ#when-should-i-torch_warn-vs-torch_warn_once)

Reply:
It's odd to me that this warning appears in loss.h. This means the warning will only be thrown in C++, but not in Python, right? Why would we do that? 

Also, why is Huber Loss especially concerned about broadcasting? Most of our functions that broadcast are happy to do it.

Reply:
I'm good with removing the warnings on both the Python and C++ side. This was done to match Smooth L1, but perhaps it's an outdated thing to do?

Reply:
Let's ditch it.

Reply:
Based on Alban's comment regarding the purpose of these warnings, I'm choosing to keep them.